update-compose.yaml

```diff
@@ -0,0 +1,8 @@
+# Config file for `updatecli compose ...`.
+# https://www.updatecli.io/docs/core/compose/
+policies:
+  - name: Handle default oblt-cli version
+    policy: ghcr.io/elastic/oblt-updatecli-policies/oblt-cli/version:0.0.2@sha256:dd79537aa64a28c57da351182a70a2a798e9b6c5eb55c243fdbd4f5bffe5133d
+    values:
+      - .ci/updatecli/values.d/scm.yml
+      - .ci/updatecli/values.d/oblt-cli.yml
```

Comment:
Can you also add support for:

- https://github.com/elastic/apm-agent-php/pull/1193

Reply:
https://github.com/elastic/oblt-actions/pull/83/commits/31103f512408ae5d0df29908028fb0337a405d42